buffer/src/test/java/io/netty/buffer/ByteBufDerivationTest.java

```diff
@@ -21,8 +21,8 @@
 import java.nio.ByteOrder;
 import java.util.Random;
 
+import static org.hamcrest.MatcherAssert.assertThat;
 import static org.hamcrest.Matchers.*;
-import static org.junit.Assert.*;
```

Comment:
What's our preference on star-imports? I thought we were avoiding them everywhere. If so, we could "organise import" on the files already touched by this PR.

Reply:
I actually would like to remove these but I would do this in a followup pr

Reply:
Works for me 👍